%YAML:1.0
---
Calibration Timestamp: "2020-05-13 17:41:03"
Camera Matrix: !!opencv-matrix
   rows: 3
   cols: 3
   dt: d
   data: [ 4.9884787297112337e+02, 0., 3.2162260564768673e+02, 0.,
       4.9992191245330628e+02, 2.4500401609096454e+02, 0., 0., 1. ]
distCoeffs: !!opencv-matrix
   rows: 1
   cols: 5
   dt: d
   data: [ 2.0596522890042857e-01, -5.0903156752563228e-01,
       1.6316369721863565e-03, 1.6653581376871636e-03,
       3.0624561006314766e-01 ]
Re-projection Error: 1.1611812561750412e-01
